### PR TITLE
Empty print statement

### LINKED ISSUE #179: Empty print should be allowed
`print` by itself is a valid line of Brightscript that can be used to add a new line. Doing this throws an error ```Found unexpected token '
'```

### PR DESCRIPTION
**Pull request by TwitchBronBron**
Fixes #179

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "clean": "rimraf ./lib ./types",
     "test": "jest",
     "lint": "tslint --project .",
-    "prettier:write": "prettier --write '{bin,src,test}/**/*.{js,ts}'",
-    "prettier": "prettier --check '{bin,src,test}/**/*.{js,ts}'",
+    "prettier:write": "prettier --write \"{bin,src,test}/**/*.{js,ts}\"",
```
Comment (sjbarag) [inline package.json]: Any idea what happened with this change?  Doesn't seem like it's necessary 😃

File: src/parser/Parser.ts [hunk 2]
```diff
@@ -1056,7 +1056,14 @@ export class Parser {
                 | Expr.Expression
                 | Stmt.PrintSeparator.Tab
                 | Stmt.PrintSeparator.Space)[] = [];
-            values.push(expression());
+
+            //print statements can be empty, so look for empty print conditions
+            if (isAtEnd() || check(Lexeme.Newline, Lexeme.Colon)) {
+                var emptyStringLiteral = new Expr.Literal(new Int32(1), printKeyword.location);
```
Comment (sjbarag) [inline src/parser/Parser.ts]: I'm a little confused – won't this print `1` every time?  It doesn't seem like an empty string 😃

Comment (sjbarag) [reply src/parser/Parser.ts]: Dangit github updated while I was reviewing

File: src/parser/Parser.ts [hunk 3]
```diff
@@ -1056,7 +1056,14 @@ export class Parser {
                 | Expr.Expression
                 | Stmt.PrintSeparator.Tab
                 | Stmt.PrintSeparator.Space)[] = [];
-            values.push(expression());
+
+            //print statements can be empty, so look for empty print conditions
+            if (isAtEnd() || check(Lexeme.Newline, Lexeme.Colon)) {
+                var emptyStringLiteral = new Expr.Literal(new BrsString(""), printKeyword.location);
```
Comment (sjbarag) [inline src/parser/Parser.ts]: Mind switching this to a `let` real quick?  I could have sworn I had `tslint` configured to disallow `var` 🤔

File: package.json (hunk 1 above)
Comment (sjbarag) [reply package.json]: Ah, mentioned in another PR: https://github.com/sjbarag/brs/pull/247#discussion_r288729479

Nice catch!

File: src/parser/Parser.ts (hunk 3 above)
Comment (TwitchBronBron) [reply src/parser/Parser.ts]: @sjbarag done.

**Pull request merged**